kong/plugins/datadog/handler.lua

```diff
@@ -48,8 +48,9 @@ local function compose_tags(service_name, status, consumer_id, tags, conf)
 end
 
 
-local function get_queue_id(conf)
-  return conf.__key__
+-- we use plugin instance ID as the queue ID
```

Comment:
Please remove this function and invoke `kong.plugin.get_id()` directly.  The comment will not be needed then, and the code will be easier to understand.